ChatQnA/ui/react/.env

```diff
@@ -1,2 +1,4 @@
 VITE_BACKEND_SERVICE_ENDPOINT=http://backend_address:8888/v1/chatqna
-VITE_DATA_PREP_SERVICE_URL=http://backend_address:6007/v1/dataprep
\ No newline at end of file
+VITE_DATA_PREP_SERVICE_URL=http://backend_address:6007/v1/dataprep
```

Comment:
We use nginx to forward messages from the frontend to the backend, just need to provide endpoint name here.

Reply:
So the env should be set as sth like this?
```
VITE_DATA_PREP_SERVICE_URL=/v1/dataprep
VITE_DATAPREP_GET_FILE_ENDPOINT=/v1/dataprep/get_file
VITE_DATAPREP_DELETE_FILE_ENDPOINT=/v1/dataprep/delete_file
```

Reply:
> So the env should be set as sth like this?
> 
> ```
> VITE_DATA_PREP_SERVICE_URL=/v1/dataprep
> VITE_DATAPREP_GET_FILE_ENDPOINT=/v1/dataprep/get_file
> VITE_DATAPREP_DELETE_FILE_ENDPOINT=/v1/dataprep/delete_file
> ```

Yes, Only need the API. Please refer to the README to start the service using the corresponding method and ensure the current code works without issues.